packages/jest-jasmine2/src/index.js

```diff
@@ -69,7 +69,32 @@ function jasmine2(config, environment, moduleLoader, testPath) {
       moduleLoader.requireModule(null, config.setupTestFrameworkScriptFile);
     }
   });
+
+  const iterableEquality = (left, right) => {
+    let result; // result needs to be `undefined` if not an iterator
```

Comment:
`if it is not an iterator.`

don't forget the full stop and please put the comment on top instead of to the right :)